core/src/main/scala/akka/persistence/r2dbc/query/javadsl/R2dbcReadJournal.scala

```diff
@@ -54,15 +68,23 @@ final class R2dbcReadJournal(delegate: scaladsl.R2dbcReadJournal)
   override def currentEventsByPersistenceId(
       persistenceId: String,
       fromSequenceNr: Long,
-      toSequenceNr: Long): Source[EventEnvelope, NotUsed] =
+      toSequenceNr: Long): Source[ClassicEventEnvelope, NotUsed] =
     delegate.currentEventsByPersistenceId(persistenceId, fromSequenceNr, toSequenceNr).asJava
 
   override def eventsByPersistenceId(
       persistenceId: String,
       fromSequenceNr: Long,
-      toSequenceNr: Long): Source[EventEnvelope, NotUsed] =
+      toSequenceNr: Long): Source[ClassicEventEnvelope, NotUsed] =
```

Comment:
Since we are taking the path of having `typed.EventEnvelope` we could also make this plugin only support the new typed queries. 
Anyway, just an idea for later. It will require us to bring all the typed queries into akka/akka first.

Reply:
It's mostly `eventsByPersistenceId` and `currentEventsByPersistenceId` we are talking about, right? I think we still want to have some of the "old" to be able to abstract over the plugin. For example to use `currentEventsByPersistenceId` when switching between Spanner and R2DBC. Another example is that ReplicatedEventSourcing is using `eventsByPersistenceId`.